legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/handler/legend/LegendResultSet.java

```diff
@@ -61,6 +61,14 @@ public Object getObject(int i) throws Exception
     {
         LegendColumn legendColumn = columns.get(i - 1);
         String value = currentRow.get(i - 1);
+        if (value.equals("null"))
+        {
+            return null;
+        }
+        if (value.equals("") && !legendColumn.getType().equals("String"))
```

Comment:
Is there a constant / enum that can be reference for the column type of "String"?